.travis.yml

```diff
@@ -31,11 +31,35 @@ matrix:
         - os: osx
           language: generic
           env: TERRYFY_PYTHON='macpython 3.6.0'
+        - services:
+            - docker
+          env: DOCKER_IMAGE=quay.io/pypa/manylinux1_x86_64
+          before_install:
+            - if ! [[ $TRAVIS_TAG ]]; then exit 0; fi
+          install:
+            - docker pull $DOCKER_IMAGE
+          script:
+            - docker run --rm -v `pwd`:/io $DOCKER_IMAGE /io/.manylinux-install.sh
+          after_success:
+            - pip install twine && twine upload -u datamade.wheelbuilder -p $PYPIPASSWORD wheelhouse/*
+            - exit 0
+        - services:
+            - docker
+          env: DOCKER_IMAGE=quay.io/pypa/manylinux1_i686
+          before_install:
+            - if ! [[ $TRAVIS_TAG ]]; then exit 0; fi
+          install:
+            - docker pull $DOCKER_IMAGE
+          script:
+            - docker run --rm -v `pwd`:/io $DOCKER_IMAGE /io/.manylinux-install.sh
+          after_success:
+            - pip install twine && twine upload -u datamade.wheelbuilder -p $PYPIPASSWORD wheelhouse/*
+            - exit 0
+  
 before_install:
     - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then git clone https://github.com/MacPython/terryfy; fi
     - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then source terryfy/travis_tools.sh; fi
     - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then get_python_environment $TERRYFY_PYTHON venv; fi
-    - if [[ "$TERRYFY_PYTHON" == "homebrew 3" ]]; then alias pip=`which pip3` ; fi
```

Comment:
This is dead code because the current travis builds does not use "homebrew 3".

This is not really related to the real purpose of this PR, so if you want it be pulled out, let me know